cmd/artesia/app/server.go

```diff
@@ -26,6 +27,28 @@ func NewServer() (*Server, error) {
 	mux := mux.NewRouter()
 	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
 		json.NewEncoder(w).Encode([]string{"Hello", "World"})
+	}).Methods("GET")
+	// Register routes for our app
+	mux.HandleFunc("/api/v1/instance", controllers.GetCurrentInstance).Methods("GET")
+	mux.HandleFunc("/api/v1/apps", controllers.RegisterOAuthApplication).Methods("POST")
+
+	// Register error handlers
+	mux.NotFoundHandler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Could we move this into a `utils.go` file under `controllers/` here? If you don't want to do that in this stubbing PR, I can add it in in mine, but I think it would be great to hold these in a separate file and just refer to them in the top-level.

Reply:
I like moving the handlers out of this file. Not a fan of things named `utils`,though. How does moving them into files like `controllers/not_found.go` and `controllers/method_not_allowed.go` sound?

Reply:
Sounds fine to me.